sov-modules/sov-modules-impl/prover-incentives/src/tests.rs

```diff
@@ -0,0 +1,195 @@
+use sov_modules_api::{default_context::DefaultContext, Address, Hasher, Module, ModuleInfo, Spec};
+use sov_state::{ProverStorage, WorkingSet};
```

Comment:
Is there a test, if proves does not have enough amount bonded to process proof? I guess it might be a valid case, especially if prover is not slashed 100%

Reply:
Added a test